commercetools-models/src/test/java/io/sphere/sdk/attributestutorial/ProductTypeCreationDemoIntegrationTest.java

```diff
@@ -528,6 +529,7 @@ public void orderImportExample() throws Exception {
                 .build();
         final OrderImportDraft orderImportDraft = OrderImportDraftBuilder
                 .ofLineItems(EURO_20, OrderState.COMPLETE, asList(lineItemImportDraft))
+                .inventoryMode(InventoryMode.NONE)
```

Comment:
This is the test for product types, I would rather remove this from here and add a test in `OrderImportCommandIntegrationTest` that checks the default behaviour (NONE) when you set nothing and trying to set one in particular.